src/applications/find-forms/tests/00-required.e2e.spec.js

```diff
@@ -33,7 +33,8 @@ function runTest(browser) {
   );
   browser.axeCheck(SELECTORS.WIDGET);
 
-  const sortedForms = sortBy(stub.data, 'id');
+  const validForms = stub.data.filter(form => form.attributes.validPdf);
```

Comment:
Just confirming here, but it looks like `stub.data` has changed from an Object to an Array, is that correct/intended? 🙂Will `stub.data` ever be `undefined`? If so we may want to do some `?.` here

Reply:
I think `.data` was an array before but it was written kinda cleverly in the `sortBy`. Am I reading it wrong? (not a _ expert 🙂 )

It's a static JSON blob so I'd like the e2e test fail if something is missing

Reply:
Oh you're right! It's always been an array, I just read it wrong 🤪